patch/gfx6/chip/llpcGfx6ConfigBuilder.cpp

```diff
@@ -109,7 +109,11 @@ Result ConfigBuilder::BuildPipelineVsFsRegConfig(
 
     const uint32_t stageMask = pContext->GetShaderStageMask();
 
-    uint64_t hash64 = 0;
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 35
+    ShaderHash hash = {};
```

Comment:
It seems uint64_t hash = {} can pass on VC.  which compiler can't support it?
if all compilers can support it, please remove #if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >=35.